.gitignore

```diff
@@ -159,3 +159,5 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
+search_roadmap_export/config.py
```

Comment:
What is this and why are we ignoring it?

Reply:
It was for a config file that has since been deprecated, thanks for spotting this :)